app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Code Style Issue: Consider using parentheses for importing multiple classes/modules to improve readability and maintain consistency with the code style guide.